ts/session/utils/libsession/libsession_utils_user_groups.ts

```diff
@@ -140,8 +138,8 @@ async function insertGroupsFromDBIntoWrapperAndRefresh(
         expirationMode: foundConvo.get('expirationMode') || 'off',
         expireTimer: foundConvo.get('expireTimer') || 0,
         displayNameInProfile: foundConvo.getRealSessionUsername(),
-        encPubkeyHex: encryptionKeyPair?.publicHex || '',
-        encSeckeyHex: encryptionKeyPair?.privateHex || '',
+        encPubkeyHex: '',
```

Comment:
I assume everything loads fine for read-only even though we no longer use the keypair?

Reply:
indeed